eks-worker-al2-variables.json

```diff
@@ -22,7 +22,7 @@
     "launch_block_device_mappings_volume_size": "4",
     "pause_container_version": "3.5",    
     "pull_cni_from_github": "true",
-    "remote_folder": "",
+    "remote_folder": "/tmp",
```

Comment:
Do we need to set this default value?

Reply:
Are we going to document what these variables actually mean and what they're for?

Reply:
`remote_folder` is a packer thing, it's the directory that packer will upload shell provisioner scripts to. It's *not* the `$PWD` when the script is actually executed, but that's an aside. https://developer.hashicorp.com/packer/docs/provisioners/shell#remote_folder

The default from packer is `/tmp`, which is what we used prior to this change. But because we expand the value (in this PR), our input should be non-blank.

Reply:
> Are we going to document what these variables actually mean and what they're for?

Yeah, that's been needed for a while. I guess I can go ahead and start the doc, and we can backfill other template variables later.